samples/.NET/MSTest/.runsettings

```diff
@@ -0,0 +1,25 @@
+<RunSettings>
+  <!-- MSTest adapter -->  
+  <MSTest>
+    <Parallelize>
+      <Workers>20</Workers>
```

Comment:
A comment would help here for users to adjust this value.

Reply:
Done